geozero/src/csv/csv_writer.rs

```diff
@@ -0,0 +1,484 @@
+        }
+        fn curvepolygon_end(&mut self, idx: usize) -> Result<()> {
+            self.wkt_writer().curvepolygon_end(idx)
+        }
+        fn multicurve_begin(&mut self, size: usize, idx: usize) -> Result<()> {
+            self.wkt_writer().multicurve_begin(size, idx)
+        }
+        fn multicurve_end(&mut self, idx: usize) -> Result<()> {
+            self.wkt_writer().multicurve_end(idx)
+        }
+        fn multisurface_begin(&mut self, size: usize, idx: usize) -> Result<()> {
+            self.wkt_writer().multisurface_begin(size, idx)
+        }
+        fn multisurface_end(&mut self, idx: usize) -> Result<()> {
+            self.wkt_writer().multisurface_end(idx)
+        }
+        fn triangle_begin(&mut self, tagged: bool, size: usize, idx: usize) -> Result<()> {
+            self.wkt_writer().triangle_begin(tagged, size, idx)
+        }
+        fn triangle_end(&mut self, tagged: bool, idx: usize) -> Result<()> {
+            self.wkt_writer().triangle_end(tagged, idx)
+        }
+        fn polyhedralsurface_begin(&mut self, size: usize, idx: usize) -> Result<()> {
+            self.wkt_writer().polyhedralsurface_begin(size, idx)
+        }
+        fn polyhedralsurface_end(&mut self, idx: usize) -> Result<()> {
+            self.wkt_writer().polyhedralsurface_end(idx)
+        }
+        fn tin_begin(&mut self, size: usize, idx: usize) -> Result<()> {
+            self.wkt_writer().tin_begin(size, idx)
+        }
+        fn tin_end(&mut self, idx: usize) -> Result<()> {
+            self.wkt_writer().tin_end(idx)
+        }
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use crate::ProcessToCsv;
+    use serde_json::json;
+
+    #[test]
+    fn geojson_to_csv() {
+        let input_geojson = json!({
+            "type": "FeatureCollection",
+            "name": "",
+            "features": [
+               {
+                   "type": "Feature",
+                   "geometry": {
+                       "type": "Point",
+                       "coordinates": [-122.329051, 47.6069]
+                   },
+                   "properties": {
+                       "address": "904 7th Av",
+                       "type": "Car Fire",
+                       "datetime": "05/22/2019 12:55:00 PM",
+                       "incident number": "F190051945"
+                   }
+               },
+               {
+                   "type": "Feature",
+                   "geometry": {
+                       "type": "Point",
+                       "coordinates": [-122.266529, 47.515984]
+                   },
+                   "properties": {
+                       "address": "9610 53rd Av S",
+                       "type": "Aid Response",
+                       "datetime": "05/22/2019 12:55:00 PM",
+                       "incident number": "F190051946"
+                   }
+               }
+            ]
+        });
+
+        let expected_output = r#"geometry,address,datetime,incident number,type
```

Comment:
note the column order here is different from what's listed in the geojson. By default geojson doesn't keep it's properties ordered. The underlying serde_json crate does offers this as an optional feature. I opened an issue to discuss exposing it in geojson here: https://github.com/georust/geojson/issues/187

I'm not totally sure we'd want to use it, but it might be a nice option since we have a concept of "property idx" in some data formats.